.azure/tutorials-linux.yml

```diff
@@ -8,9 +8,7 @@ jobs:
     pool: {vmImage: 'ubuntu-latest'}
 
     variables:
-      QISKIT_SUPPRESS_PACKAGING_WARNINGS: Y
       PIP_CACHE_DIR: $(Pipeline.Workspace)/.pip
-      QISKIT_CELL_TIMEOUT: 300
```

Comment:
I couldn't find what the deleted variables were used for. Didn't show up in search.

Reply:
You (accidentally?) removed it in Qiskit/qiskit-tutorials#1459.

Reply:
For posterity: my comment above is referring to `QISKIT_CELL_TIMEOUT`.  `QISKIT_SUPPRESS_PACKAGING_WARNINGS` disappeared in #5619.